packages/eslint-plugin/tests/rules/prefer-includes.test.ts

```diff
@@ -234,6 +234,33 @@ ruleTester.run('prefer-includes', rule, {
       `,
       errors: [{ messageId: 'preferStringIncludes' }],
     },
+    // test SequenceExpression
```

Comment:
[Non-actionable] More and more do I wish JavaScript just didn't have SequenceExpression 🙃 it's so confusing...